CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+#### 0.29.9 - 2018-07-18
+
+   * Updated usage of Dart SDK constants for Dart v2.
+     Allows users of `pkg:quiver` to support Dart 2 gold without addressing
+     all of the breaking changes in `pkg:quiver` v2.
```

Comment:
quiver 2.0 is commit-for-commit identical to quiver 1.0 other than changes to update for Dart 2.